### PR TITLE
added feature for not running window.setSize when option mobile is provided

### PR DESCRIPTION
**Pull request by abhagupta**
PR for https://github.com/sitespeedio/browsertime/issues/401
@soulgalore @tobli

### CODE REVIEW

File: lib/core/seleniumRunner.js [hunk 1]
```diff
@@ -62,26 +62,29 @@ class SeleniumRunner {
         });
       })
       .tap(() => {
-        let viewPort = this.options.viewPort;
-        if (viewPort) {
-          if (viewPort !== 'maximize') {
-            viewPort = viewPort.split('x');
-          }
-          const window = this.driver.manage().window();
+        let isChromeRunningOnMobile = this.options.mobile;
```
Comment (soulgalore) [inline lib/core/seleniumRunner.js]: Sorry the code hard to read. The right cli option is **chrome.android.package** then we know we are on Android (you cannot run on Android without or ... do you have special hack, maybe it is possible?). You can use lodash.get to get it (we are inconsistent in the code, but that is the right way 2 do it).

**Comment by soulgalore** [comment]
@abhagupta if we release 2.0 next week will you pick that up or wait to December (meaning we should release next 1.x this weekend you can use it next week?)?

**Comment by abhagupta** [comment]
HI @soulgalore  Yes, I am running these tests on saucelabs devices which connect via appium url.  In this situation none of these are available :   
   ```this.options.chrome &&
      this.options.chrome.android &&
      this.options.chrome.android.package ```.    I will keep looking if I can find other way to avoid addition of this code, but I don't think the check for above option values will help us.

**Comment by soulgalore** [comment]
Ok, so how do you set it up? You need to write  that blog post someday so we know how it works :)

**Comment by abhagupta** [comment]
Thanks @soulgalore  for getting back.  https://medium.com/@abhaguptatweet/run-sitespeed-io-on-android-devices-4777b405faaf    - still polishing but should give you details on how to run the test. Please feel free to provide feedback/mistakes/suggestions on blog as well.

**Comment by abhagupta** [comment]
@soulgalore  I dont need this change anymore.  we have extended the webdriver and overridden the `setPosition` and `setSize` functions.   I will close this PR

**Comment by soulgalore** [comment]
@abhagupta aha ok cool. thanks for sharing the post, looking forward to it!

**Pull request closed without merging**